readme.md

```diff
@@ -356,6 +356,9 @@ request(`https://${config.host}/production/`, {
 });
 ```
 
+## Aborting the request
```

Comment:
Done :3.
I don't think we should. It sounds worse, granted, but Node consistently calls it 'abort'.